tests/helpers/Participant.ts

```diff
@@ -122,14 +125,47 @@ export class Participant {
             userInfo: {
                 displayName: this._name
             }
-        }) || '';
+        };
+
+        if (context.iframeAPI) {
+            config.room = 'iframeAPITest.html';
+        }
+
+        let url = urlObjectToString(config) || '';
+
+        if (context.iframeAPI) {
+            const baseUrl = new URL(this.driver.options.baseUrl || '');
+
+            // @ts-ignore
+            url = `${this.driver.iframePageBase}${url}&domain="${baseUrl.host}"&room="${context.roomName}"`;
+
+            if (baseUrl.pathname.length > 1) {
+                // remove leading slash
+                url = `${url}&tenant="${baseUrl.pathname.substring(1)}"`;
+            }
+        }
+        if (this._jwt) {
+            url = `${url}&jwt="${this._jwt}"`;
+        }
 
         await this.driver.setTimeout({ 'pageLoad': 30000 });
 
-        await this.driver.url(url);
+        // workaround for https://github.com/webdriverio/webdriverio/issues/13956
+        if (url.startsWith('file://')) {
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Why not let it throw if there is an exception?

Reply:
It fails the test because it comes with a timeout. But the problem is fixed and will be available on the next release. I will update it once it is out.

Reply:
👍